### PR TITLE
Add XeroLinux

### PR DESCRIPTION
**Pull request by zen0bit**
[XeroLinux](https://xerolinux.xyz/)

Initial commit..

As you asked..
Test pls. @TechXero

### CODE REVIEW

**Review by philclifford — APPROVED** [review]
LGTM

**Comment by flexiondotorg** [comment]
Please rebase against `master`.

**Comment by zen0bit** [comment]
done

File: quickget [hunk 1]
```diff
@@ -76,7 +76,8 @@ function pretty_name() {
     ubuntu-unity)       PRETTY_NAME="Ubuntu Unity";;
     vanillaos)          PRETTY_NAME="Vanilla OS";;
     void)               PRETTY_NAME="Void Linux";;
-    vxlinux)            PRETTY_NAME="VX Linux";;
+    vxlinux)            PRETTY_NAME="VX linux";;
```
Comment (philclifford) [inline quickget:79]: That should be left as "VX Linux"

https://github.com/dessington/vx-linux/blob/d68b13d0a73afc7b8f3ef9649cfd17668947adca/README.md?plain=1#LL1C1-L1C1

**Review by philclifford — CHANGES_REQUESTED** [review]

**Review by philclifford — APPROVED** [review]

**Pull request merged**